packages/dictionary/src/metaSchema/dictionarySchemas.ts

```diff
@@ -17,17 +17,18 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { z as zod } from 'zod';
+import { z as zod, type ZodType } from 'zod';
```

Comment:
COOL! hadn't seen this done that way

Reply:
Yeah its nicer like this. You can set VSCode to automatically write type imports like this when it is adding import statements for you.

Reply:
awesome! gotta share the how sometime this week over zoom/slack